packages/cli/src/commands/deploy/index.ts

```diff
@@ -79,5 +105,30 @@ export default async client => {
     }
   }
 
-  return deploy(client, paths, localConfig, args);
+  return deploy(client, paths, localConfig, {
+    '--force': Boolean,
+    '--with-cache': Boolean,
+    '--public': Boolean,
+    '--no-clipboard': Boolean,
+    '--env': [String],
+    '--build-env': [String],
+    '--meta': [String],
+    // This is not an array in favor of matching
+    // the config property name.
+    '--regions': String,
+    '--prod': Boolean,
+    '--confirm': Boolean,
+    '-f': '--force',
+    '-p': '--public',
+    '-e': '--env',
+    '-b': '--build-env',
+    '-C': '--no-clipboard',
+    '-m': '--meta',
+    '-c': '--confirm',
+
+    // deprecated
+    '--name': String,
+    '-n': '--name',
+    '--target': String,
+  });
```

Comment:
Is there anyway to avoid duplicating these args?

Reply:
Ya, done.